src/modules/pages/NewApiPage/NewApiForm.js

```diff
@@ -31,11 +32,13 @@ const ApiForm = (props) => {
     const {
         handleSubmit,
         initialValues,
+        apiSchema,
         plugins,
         excludePlugin,
         selectPlugin,
         selectedPlugins,
     } = props;
+    console.error('NEW API', props);
     const parse = value => (value === undefined ? undefined : parseInt(value));
```

Comment:
`const parse = value => value && parseInt(value);`

Reply:
Damn... you are right! Don't understand why I did do that by myself 😞